integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/preaggregate/TestPreAggCreateCommand.scala

```diff
@@ -463,6 +464,39 @@ class TestPreAggCreateCommand extends QueryTest with BeforeAndAfterAll {
     executorService.shutdown()
   }
 
+  test("support set carbon.query.directQueryOnDataMap.enabled=true") {
+    val rootPath = new File(this.getClass.getResource("/").getPath
+      + "../../../..").getCanonicalPath
+    val testData = s"$rootPath/integration/spark-common-test/src/test/resources/sample.csv"
+    sql("drop table if exists mainTable")
+    sql(
+      s"""
+         | CREATE TABLE mainTable
+         |   (id Int,
+         |   name String,
+         |   city String,
+         |   age Int)
+         | STORED BY 'org.apache.carbondata.format'
+      """.stripMargin);
+
+
+    sql(
+      s"""
+         | LOAD DATA LOCAL INPATH '$testData'
+         | into table mainTable
+       """.stripMargin);
```

Comment:
for scala code semi-colon `;` is not required

Reply:
ok, removed